constants.py

```diff
@@ -0,0 +1,11 @@
+# This is the constant file that is shared with both python and shell scripts
+# in seconds
+TIME_TO_RECORD=30
+AUDIO_CAPTURE_REMOTE_PORT=7777
+MEDIA_DIR="/missed_moment_media"
+AUDIO_CAPTURE_TEMP_FILENAME="/missed_moment_media/mm-timemachine.wav"
+VIDEO_PATTERN="missed-moment.*merged.*mp4"
+USB_MOUNT_DIR="/missed_moment_usb"
+USB_DIR="/missed_moment_usb/missed_moment"
+# number of days to keep files minus 1
+DAYS_TO_KEEP_FILES=6
```

Comment:
My mind went back and forth but ultimately I like this approach. This is especially useful if we need to put things like datetime objects in config. No need to worry about casting/converting/etc. Everything here is already is the datatype we need. 
This file can become more complicated if/when we need it to.